src/lib/device-types/build-info-facade.ts

```diff
@@ -21,6 +21,9 @@ export const getLogoUrl = memoizee(
 		buildId: string,
 	): Promise<string | undefined> => {
 		try {
+			if (!FILES_HOST) {
+				return;
+			}
```

Comment:
This can be outside the try/catch block.

Reply:
Sure